src/features/defi/providers/foxy/components/FoxyManager/Overview/Claim/ClaimStatus.tsx

```diff
@@ -88,7 +88,7 @@ export const ClaimStatus = () => {
   const feeAsset = useAppSelector(state => selectAssetById(state, feeAssetId))
   const feeMarketData = useAppSelector(state => selectMarketDataById(state, feeAssetId))
 
-  const { refetch: refetchFoxyBalances } = useFoxyBalances()
+  const { refetch: refetchFoxyBalances } = useFoxyBalances({ accountNumber: 0 })
```

Comment:
This is wrong, we're in the context of a modal route and should be able to get the data for the right accountNumber / accountId

Reply:
1d371c78a6e2f3b0c39afc25023b978286be3c46